src/qibo/gates/abstract.py

```diff
@@ -54,7 +59,15 @@ def __init__(self):
         self.device_gates = set()
         self.original_gate = None
 
-    def to_json(self):
+    @property
+    def raw(self) -> dict:
```

Comment:
May I suggest to rename this to `to_dict` as done for other python packages?

Reply:
See above